### PR TITLE
Edits to Introduction and bios

### PR DESCRIPTION
**Pull request by amyrbrown**
I fixed a few little things in the introduction and bios. More notes on the File Changed page. (I really like the Introduction!)

### CODE REVIEW

File: minutiae/intro.tex [hunk 1]
```diff
@@ -1,30 +1,30 @@
 \begin{aosachapter}{Introduction}{s:intro}{Michael DiBernardo}
 
 This is the fourth volume in the \emph{Architecture of Open Source Applications}
-series, and the first to not feature the word `architecture' anywhere in the
+series, and the first to not feature the words `open source applications' anywhere in the
```
Comment (amyrbrown) [inline minutiae/intro.tex]: The last one was *The Performance of Open Source Applications*.

File: minutiae/intro.tex [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
+few thousand lines of code, so, while big problems can be interesting to
 read about, they can also be challenging to learn from.
 
 \emph{500 Lines or Less} focuses on the design decisions that programmers make
-in the small when they are building something new. The programs you will read
-in this book were all written from scratch for this purpose, although several
-of them were inspired by larger projects that the authors had worked on before.
+in the small when they are building something new. The programs you will read about
+in this book were all written from scratch for this purpose (although several
+of them were inspired by larger projects that the authors had worked on previously).
```
Comment (amyrbrown) [inline minutiae/intro.tex]: "Before" is more concise, but you had two befores in a row.

File: minutiae/intro.tex [hunk 3]
```diff
@@ -83,13 +83,13 @@ \section*{Contributors}
...
 
 \emph{Audrey Tang (Spreadsheet)}: A self-educated programmer and translator, Audrey works with Apple as an independent contractor on cloud service localization and natural language technologies. Audrey has previously designed and led the first working Perl 6 implementation, and served in computer language design committees for Haskell, Perl 5, and Perl 6. Currently Audrey is a full-time g0v contributor and leads Taiwan’s first e-Rulemaking project.
 
-\emph{Leah Hanson (Static analysis)}: Leah Hanson is a proud alumni of Hacker School and loves helping people learn about Julia. She blogs at \url{blog.leahhanson.us} and tweets at \url{@astrieanna}.
+emph{Leah Hanson (Static analysis)}: Leah Hanson is a proud alum of Hacker School and loves helping people learn about Julia. She blogs at \url{blog.leahhanson.us} and tweets at \url{@astrieanna}.
```
Comment (amyrbrown) [inline minutiae/intro.tex]: **Alumni** is plural, but it turns out **alumnus** is masculine, and **alumna** just sounds too precious, so I went with the more modern and casual **alum**.

File: minutiae/intro.tex [hunk 4]
```diff
@@ -112,7 +112,7 @@ \section*{Acknowledgments}
 Borys Pierov \\
 Carise Fernandez \\
 Charles	Stanhope \\
-Chris Atlee \\
+Chris AtLee \\
```
Comment (amyrbrown) [inline minutiae/intro.tex]: Can you double-check all the other names?

Comment (MichaelDiBernardo) [inline minutiae/intro.tex]: I actually triple-checked them, so if I haven't gotten it right by now I don't think I'm going to do any better :)

**Pull request merged**